helm-charts/sawmills-collector/tests/haproxy_probe_config_test.yaml

```
@@ -210,6 +234,44 @@ tests:
+  - it: should route load balancer main collector readiness through custom haproxy readiness path
+    template: load-balancer.yaml
+    values:
+      - ../values.yaml
+    set:
+      loadBalancer.enabled: true
+      haproxy.enabled: true
+      haproxy.probes.readiness.path: /lb-ready
+      haproxy.probes.readiness.port: 13135
+      haproxy.mapping.logs_http_10000.from: 10000
+      haproxy.mapping.logs_http_10000.to.port: 10518
+      haproxy.mapping.logs_http_10000.to.protocol: "TCP"
```

YAMLlint (1.38.0)

[error] string value is redundantly quoted with any quotes

(quoted-strings)